config/initializers/session_store.rb

```diff
@@ -1,3 +1,3 @@
 # Be sure to restart your server when you modify this file.
 
-FinderFrontend::Application.config.session_store :disabled
+FinderFrontend::Application.config.session_store :cookie_store
```

Comment:
Just as a sanity check, Rails isn't going to start setting any cookies unless people start visiting the login URL, right?

Reply:
It looks like we set a `_session_id` cookie.  We probably don't want this set unless the user logs in.

Adding this to the `ApplicationController` seems to do the trick of only enabling this cookie when the user is logged in:
```
after_action :skip_session_cookie

...

  def skip_session_cookie
    unless current_user
      request.session_options[:drop] = true
    end
  end
```